### ISSUE TITLE
OMV-NUT plugin breaks email notifications on OMV5

### ISSUE THREAD

**Issue by SadButTrueAT** [openmediavault/openmediavault#448]
### Description of issue/question
after installing the OMV5-NUT plugin email notifications and setup of email notifications are broken.

### Steps to reproduce issue
installing the NUT plugin on OMV5 change something on the notification page and the notifications setup ends up in an apply bar loop.

after removing the plugin notifications are working again as ever 

https://forum.openmediavault.org/index.php/Thread/28594-OMV5-setup-notifications-don-t-work/?postID=213378#post213378

**Comment by votdev** [comment]
I can reproduce this behaviour sometimes. Sending an email fails sometimes, but then sending another email some seconds later it works again.

```
status=deferred (Host or domain name not found. Name service error for name=mail.foo.bar type=MX: Host not found, try again)
```

To me it looks like that a driver of the NUT toolkit is causing the issue.

I will close this issue because this can't be fixed by the OMV project due this is no configuration problem.

**Comment by SadButTrueAT** [comment]
sorry votdev - i dont think its NUT related bc i have NUT manually installed and running since a few day - with your config files (without js and webiface) right out of my OMV4 backup including upsshed_cmd - everything email and NUT are working without problem i can change the email without loop, i get apt-update emails, i receive nut mails etc... everything is working as expected even collectd is working with nut configured!

i think there is something other broken than nut toolkit... but i have found a solution for myself so it's ok

**Comment by PrplHaz4** [comment]
I'm seeing this same behavior as well.  Removing NUT 5.0-1 plugin now allows me to save and apply changes to smtp in the notification screen (as well as successfully sending email).

The log entries that led me here were:
```
systemd[1]: /lib/systemd/system/nut-monitor.service:6: PIDFile= references path below legacy directory /var/run/, updating /var/run/nut/upsmon.pid → /run/nut/upsmon.pid; please update the unit file accordingly.

omv-engined[22245]: PHP Fatal error: Uncaught Error: Call to a member function preDeploy() on null in /usr/share/openmediavault/engined/rpc/config.inc:161
```
The PHP preDeploy() error seemed to occur every time I hit the "Apply" button on the notification screen (with NUT enabled).

**Comment by votdev** [comment]
Fixed in openmediavault 5.0.12.
